package.json

```diff
@@ -12,10 +12,10 @@
     "hexo-generator-category": "^0.1.2",
     "hexo-generator-index": "^0.2.0",
     "hexo-generator-tag": "^0.2.0",
-    "hexo-renderer-ejs": "^0.2.0",
+    "hexo-renderer-ejs": "0.3.1",
```

Comment:
Not ^ for these deps?

Reply:
Good eye, and good question.  This is to accommodate [Renovate](renovateapp.com) (bot), which I'll add to this repo soon.